docker_client.go

```diff
@@ -97,7 +97,7 @@ func splitDockerImage(img string) (string, string, string) {
 func getContainers(client *docker.Client) ([]*RuntimeContainer, error) {
 
 	apiContainers, err := client.ListContainers(docker.ListContainersOptions{
-		All:  false,
+		All:  true,
```

Comment:
It seems like a `Generator` type would pretty naturally fall out of the effort of turning `docker-gen` into a library that I mentioned over in https://github.com/jwilder/nginx-proxy/pull/254#issuecomment-152549378